kie-dmn-feel/src/main/antlr4/org/kie/dmn/feel/parser/feel11/FEEL_1_1.g4

```diff
@@ -664,7 +664,6 @@ LE : '<=';
 GE : '>=';
 NOTEQUAL : '!=';
 
-QUESTION : '?';
```

Comment:
A variable in FEEL can start with a '?' character and this lexer rule was clashing with the identifier rule and causing a bug. This symbol is not used for anything else in FEEL, so this Lexer rule can (and has to) be removed.